tests/integ/timeout.py

```diff
@@ -46,7 +46,12 @@ def timeout(seconds=0, minutes=0, hours=0):
 
 @contextmanager
 def timeout_and_delete_endpoint_by_name(
-    endpoint_name, sagemaker_session, seconds=0, minutes=45, hours=0
+    endpoint_name,
+    sagemaker_session,
+    seconds=0,
+    minutes=45,
+    hours=0,
+    sleep_between_cleanup_attempts=10,
```

Comment:
Added this as a parameter to prevent the unit tests from taking an addition 10*3*2=60 seconds.